.github/workflows/ci.yml

```diff
@@ -1,16 +1,81 @@
 name: CI
+
 on:
   push:
-    branches: [main, release]
+    branches: [main, release/*]
   pull_request:
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+    branches: [main]
```

Comment:
Should this actions be invoked only in main branch?
I think we should run some jobs on every pull request.

Reply:
You're right, I'm thinking `ci-web`, `ci-server` and `slack-notifictation` will do. What do you think?

Reply:
It looks good!

Reply:
https://github.com/reearth/reearth/pull/393/commits/1881b468bec719a61cf1ed4792ae8cfc10289470 should resolve this.